crates/figma_import/src/transform_flexbox.rs

```diff
@@ -1109,24 +1110,25 @@ fn visit_node(
     // we need to set our size depending on whether the size policy is hug or fixed.
     if is_widget_list {
         if size_policy == SizePolicy::Hug {
-            style.layout_style.position_type = PositionType::Relative.into();
-            style.layout_style.width = DimensionProto::new_auto();
-            style.layout_style.height = DimensionProto::new_auto();
+            style.layout_style_mut().position_type = PositionType::Relative.into();
+            style.layout_style_mut().width = DimensionProto::new_auto();
+            style.layout_style_mut().height = DimensionProto::new_auto();
         } else {
-            style.layout_style.position_type = PositionType::Absolute.into();
+            style.layout_style_mut().position_type = PositionType::Absolute.into();
         }
     }
 
     // Blend mode is common to all elements.
-    style.node_style.blend_mode = convert_blend_mode(node.blend_mode).into();
+    style.node_style_mut().blend_mode = convert_blend_mode(node.blend_mode).into();
 
-    style.node_style.stroke.as_mut().map(|stroke| {
+    style.node_style_mut().opacity = if node.opacity < 1.0 { Some(node.opacity) } else { None };
```

Comment:
Why did this line get pulled out of the `map()` below?

Reply:
Because the line was trying to take a mutable reference to node_style within a block that was already working on a mutable reference to node_style. It was giving a compile error and moving it out solved it. It didn't seem like there would be any issue, it seems unconnected to everything else in the map operation.